proto/lbm/fswap/v1/query.proto

```diff
@@ -19,7 +19,10 @@ service Query {
 
 message QuerySwappedRequest {}
 message QuerySwappedResponse {
-  Swapped swapped = 1 [(gogoproto.nullable) = false];
+  cosmos.base.v1beta1.Coin old_coin_amount = 1
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/Finschia/finschia-sdk/types.Coin"];
+  cosmos.base.v1beta1.Coin new_coin_amount = 2
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/Finschia/finschia-sdk/types.Coin"];
```

Comment:
`old_coin_amount` in `message Swapped` has different type to the one in here. 
This is a little bit confusing.

Reply:
I will change the name in message `EventSwapCoins` and `QueryXXXResponse`
`new/old_coin_amount` will be`types.Int` and `new/old_coin` will be`types.Coin`